packages/fuel-indexer-lib/src/config/database.rs

```diff
@@ -156,7 +141,10 @@ impl FromStr for DatabaseConfig {
                     database: database.to_string(),
                 })
             }
-            _ => Err(IndexerConfigError::DbUrlParseError),
+            _ => {
+                println!("Unsupported database. Please check your database URL.");
+                Err(ParseError::InvalidIpv4Address)
```

Comment:
Is this really an `ParseError::InvalidIpv4Address` error thought? Maybe `unimplemented!()` ?

Reply:
Ooh, `unimplemented!()` is much better.